examples/volumes/vsphere/README.md

```diff
@@ -94,6 +94,11 @@
           volumePath: "[datastore1] volumes/myDisk"
           fsType: ext4
       ```
+      In the above example datastore1 not placed within any Folder or Datastore Cluster. If datastore is placed within Datastore Cluster or some folder, appropriate path should be provided in the VolumePath as below. 
```

Comment:
I don't quite follow this sentence. should it be: "datastore1 _is_ not placed.. "?

"_the_ appropriate path"?